sdk/communication/communication-phone-numbers/test/list.spec.ts

```diff
@@ -23,33 +22,12 @@ describe("PhoneNumbersClient - lists", function() {
   });
 
   it("can list all purchased phone numbers", async function() {
+    let all = 0;
     for await (const purchased of client.listPurchasedPhoneNumbers()) {
       assert.match(purchased.phoneNumber, /\+\d{1}\d{3}\d{3}\d{4}/g);
       all++;
     }
 
     assert.isTrue(all > 0);
-  }).timeout(10000);
-
-  it("can skip a phone number", async function() {
-    let countWhenSkipped = 0;
-    for await (const purchased of client.listPurchasedPhoneNumbers({ skip: 1 })) {
-      assert.match(purchased.phoneNumber, /\+\d{1}\d{3}\d{3}\d{4}/g);
-      countWhenSkipped++;
-    }
-
-    assert.isTrue(countWhenSkipped > 0);
-    assert.isTrue(countWhenSkipped < all);
-  }).timeout(10000);
-
-  // TODO: revisit when service returns nextLink
-  it("can list by page", async function() {
```

Comment:
Can we file a bug to bring this back once the service has fixed the nextLink? [cross-language bug]